bitnami/apache/values.yaml

```diff
@@ -473,3 +473,56 @@ metrics:
     ##   cpu: 100m
     ##   memory: 128Mi
     requests: {}
+  ## Prometheus Operator PodMonitor configuration
+  ##
+  podMonitor:
+    ## @param metrics.podMonitor.enabled if `true`, creates a Prometheus Operator PodMonitor (also requires `metrics.enabled` to be `true`)
+    ##
+    enabled: false
+    ## @param metrics.podMonitor.namespace Namespace for the PodMonitor Resource (defaults to the Release Namespace)
+    ##
+    namespace: ""
+    ## @param metrics.podMonitor.interval Interval at which metrics should be scraped.
+    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ## e.g:
+    ## interval: 10s
+    ##
+    interval: ""
+    ## @param metrics.podMonitor.scrapeTimeout Timeout after which the scrape is ended
+    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ## e.g:
+    ## scrapeTimeout: 10s
+    ##
+    scrapeTimeout: ""
+    ## @param metrics.podMonitor.additionalLabels Additional labels that can be used so PodMonitor will be discovered by Prometheus
+    ##
+    additionalLabels: {}
+    ## @param metrics.podMonitor.relabelings RelabelConfigs to apply to samples before scraping
+    ##
+    relabelings: []
+    ## @param metrics.podMonitor.metricRelabelings MetricRelabelConfigs to apply to samples before ingestion
+    ##
+    metricRelabelings: []
```

Comment:
Is there a reason why you decided to use a PodMonitor instead of a ServiceMonitor?

Reply:
This is what we usually use for other Bitnami charts that expose webapps. E.g. WordPress, see https://github.com/bitnami/charts/blob/master/bitnami/wordpress/values.yaml#L765

Reply:
Hi, at first time, i use podmonitor if no service is present for metrics. To match with the wordpress i have added a metrcis-svc and refactor the podmonitor to servicemonitor.